src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/SlimefunItem.java

```diff
@@ -220,6 +230,9 @@ else if (SlimefunStartup.getItemCfg().getBoolean(this.name + ".enabled")) {
 				}
 				
 				if (SlimefunStartup.getCfg().getBoolean("options.print-out-loading")) System.out.println("[Slimefun] Loaded Item \"" + this.getName() + "\"");
+			} else if (!SlimefunStartup.getItemCfg().getBoolean(this.name + ".enabled")) {
```

Comment:
It comes right after the opposite check, so you just have to write

else, no need to specify an else if